src/components/calcite-slider/calcite-slider.scss

```diff
@@ -94,13 +96,15 @@ $calcite-slider-tick-height: 4px;
   width: $calcite-slider-thumb-size;
   height: 0.75em;
   @include font-size(-3);
+  font-weight: 500;
   line-height: 1;
   color: var(--calcite-slider-label);
   text-align: center;
 }
 
 .slider__thumb:hover .slider__handle {
-  border-color: var(--calcite-slider-spot);
+  border-width: 3px;
```

Comment:
It would be nice to have an ease-in-out, default $transition on this border width adjustment. A little jumpy now.

Reply:
Additionally, should the precise handle's "line" turn the same blue on hover? It's kind of weird to have a tiny amount of dark gray sandwiched between two blues.

Reply:
Yeah, it's nice to have the handle line turn blue too. Updated.